app/model/transaction.py

```diff
@@ -0,0 +1,18 @@
+from repository.database import Base, generate_uuid, gen_now
+
+from sqlalchemy.types import CHAR, NUMERIC, DATETIME
+from sqlalchemy import Column, ForeignKey
+from decimal import Decimal
+
+
+class Transaction(Base):
+    __tablename__ = "transaction"
+
+    transaction_id = Column(CHAR(32), primary_key=True, default=generate_uuid)
+    account_id = Column(CHAR(32), ForeignKey("account.account_id"))
+    amount = Column(NUMERIC(precision=3))
```

Comment:
Here I've setted the precision to three so latter on calculations we could be also able to have complex calculations with decimals using [bankers rounding](https://stackoverflow.com/questions/45223778/is-bankers-rounding-really-more-numerically-stable) or a proper numeric storage.